packages/flame_jenny/jenny/lib/src/parse/tokenize.dart

```diff
@@ -653,9 +653,18 @@ class _Lexer {
         pushToken(Token.closeMarkupTag, position - 2);
       }
       if (tokens.last == Token.closeMarkupTag) {
-        // Self-closing markup tag such as `[img/]`: consume a single whitespace
-        // character after such tag (if present).
-        eat($space);
+        final iter = tokens.reversed
```

Comment:
iter? It's not really an iterator either, is it? If it is name it iterator.

Reply:
it's an `Iterable`, renamed.